SpiceSharpTest/Components/Semiconductors/BJT/BJTTest.cs

```diff
@@ -0,0 +1,123 @@
+using SpiceSharp;
+using SpiceSharp.Circuits;
+using SpiceSharp.Components;
+using SpiceSharp.IntegrationMethods;
+using SpiceSharp.Simulations;
+using SpiceSharpTest.Utils;
+
+namespace SpiceSharpTest.Components.Semiconductors.BJT
+{
+    [TestClass]
+    public class BJTTest
+    {
+        [TestMethod]
+        public void Emitter_Follower_DC()
+        {
+            // A test for BJT transistor (model mjd44h11) acting in an emitter-follower transistor circuit.
+            // A circuit consists of a BJT transistor and resistor connected to the transistor emitter.
+            // The voltage betwen base and emitter is around 0.7V when a transistor is in active mode.
+
+            var collectorVoltage = 100.0;
+            var baseVoltages = new double[] { 1.0, 2.0, 10.0, 20.00, 30.0, 60.0, 90.0, 98.0 };
+            var resistancesOfResistorConnectedToEmitter = new int[] { 5, 10, 30, 40, 50, 60, 70, 1000 };
+
+            foreach (var baseVoltage in baseVoltages)
+            {
+                foreach (var resistance in resistancesOfResistorConnectedToEmitter)
+                {
+                    double resistorCurrent = 0;
+
+                    var emitterVoltage =
+                        GetEmitterVoltageFromEmitterFollowerCircuit(
+                            baseVoltage,
+                            collectorVoltage,
+                            resistance,
+                            out resistorCurrent);
+
+                    var V_BE = baseVoltage - emitterVoltage;
+                    Assert.That.AreEqualWithTol(0.7, V_BE, 0, 0.5);
+                }
+            }
+        }
+
+        private static double GetEmitterVoltageFromEmitterFollowerCircuit(double baseVoltage, double collectorVoltage,
+            int resistance, out double resistorCurrent)
+        {
+            var baseVoltageSource = new Voltagesource(
+                new CircuitIdentifier("V1"),
+                new CircuitIdentifier("V_B"),
+                new CircuitIdentifier("gnd"),
+                baseVoltage);
+
+            var collectorVoltageSource = new Voltagesource(
+                new CircuitIdentifier("V2"),
+                new CircuitIdentifier("V_C"),
+                new CircuitIdentifier("gnd"),
+                collectorVoltage);
+
+            var resistor = new Resistor(new CircuitIdentifier("R"),
+                new CircuitIdentifier("V_E"),
+                new CircuitIdentifier("gnd"),
+                resistance
+            );
+
+            var bjt = new SpiceSharp.Components.BJT(new CircuitIdentifier("Q1"));
+            var model = GetBjtModel();
+            bjt.SetModel(model);
+
+            bjt.Connect(
+                new CircuitIdentifier("V_C"),
+                new CircuitIdentifier("V_B"),
+                new CircuitIdentifier("V_E"),
+                new CircuitIdentifier("gnd"));
+
+            Circuit ckt = new Circuit();
+            ckt.Objects.Add(baseVoltageSource, collectorVoltageSource, resistor, bjt);
```

Comment:
Since you are using a netlist parser for the model, consider doing the same here for for readability.

Reply:
Sure, I will change it to netlist

Reply:
I've changed the test to use netlist.